package.xml

```diff
@@ -19,6 +19,7 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
 
+  <depend>libboost-dev</depend>
```

Comment:
is there a smaller dependency by any chance?

Reply:
https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L2833-L2842

It is basic core one

If [boost](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L376-L392) is included, then it pulls in everything